package.json

```diff
@@ -33,23 +34,23 @@
 		"ws": "~7.4.0"
 	},
 	"devDependencies": {
-		"apidoc": "~0.17.0",
+		"apidoc": "^0.51.1",
 		"chai": "*",
 		"chai-http": "*",
 		"eslint": "~6.6.0",
-		"grunt": "~1.3.0",
+		"grunt": "^1.5.3",
 		"grunt-contrib-cssmin": "^3.0.0",
-		"grunt-contrib-imagemin": "^4.0.0",
-		"grunt-contrib-jshint": "~2.1.0",
-		"grunt-contrib-nodeunit": "~2.1.0",
+		"grunt-contrib-imagemin": "^2.0.1",
+		"grunt-contrib-jshint": "^3.2.0",
```

Comment:
Create a new PR if you want to upload the package.json or package-lock.json. These changes does not relate to assignment feature, so remove them from this PR.

Reply:
Okay ! Will remember in future